src/groupByKey.ts

```diff
@@ -2,6 +2,10 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: string): GroupsMap<T> {
```

Comment:
`string` type is not enough specified: use [keyof](https://www.typescriptlang.org/docs/handbook/2/keyof-types.html)